ee/agent/storage/sqlite/migrations/000001_kv.up.sqlite

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS keyvalue_pairs (
+    pair_id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
```

Comment:
Do you need this? Isn't key_name enough?

Reply:
I think I'm used to avoiding varchar primary keys for performance reasons that are not at all relevant to this table 🙂 getting rid of this, making the `key_name` the primary key, and using `WITHOUT ROWID` seems most appropriate?

Reply:
Updated like ☝️